e2e/cypress/integration/collapsed_reply_threads/crt_settings_spec.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+describe('CRT Settings', () => {
+    let testTeam;
+    let testChannel;
+
+    before(() => {
```

Comment:
The test requires specific config values from `ServiceSettings.CollapsedThreads` and/or `ServiceSettings.ThreadAutoFollow`.
I'd suggest to add the following default config values at e2e/cypress/support/api/on_prem_default_config.json & cloud_default_config.json 
```
"ServiceSettings": {
    ...
    "ThreadAutoFollow": true,
    "CollapsedThreads": "disabled"
},
```

and then explicitly set the required config per spec file using `cy.apiUpdateConfig`.

Same comment to other spec files.